include/sparta/PowersetAbstractDomain.h

```diff
@@ -235,6 +235,9 @@ class PowersetAbstractDomain
       return this->get_value()->contains(e);
     }
     }
+    SPARTA_ASSERT(false && "unknown AbstractValueKind");
```

Comment:
We need to include:
```
#include <sparta/Exceptions.h>
```
Otherwise we are getting the error:
```
xxx/sparta/PowersetAbstractDomain.h:238:5: error: use of undeclared identifier 'assert'
    SPARTA_ASSERT(false && "unknown AbstractValueKind");
    ^
```

Reply:
Fixed